presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/operator/aggregation/UnionAgg.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.geospatial.operator.aggregation;
```

Comment:
Operator means a specific thing, e.g join, aggregation, filter and projection are operator. `count`, `sum`, `avg` and `union_agg` are aggregate functions. Hence, I'd drop `.operator` from the package name.